kt/godot-library/godot-core-library/src/main/kotlin/godot/core/KtObject.kt

```diff
@@ -55,6 +55,15 @@ abstract class KtObject : NativeWrapper {
 
     protected abstract fun new(scriptIndex: Int)
 
+    open fun _get(property: StringName): Any = throw NotImplementedError("_get is not implemented for Object")
+    open fun _getPropertyList(): VariantArray<Dictionary<Any, Any>> = throw NotImplementedError("_getPropertyList is not implemented for Object")
+    open fun _propertyCanRevert(name: StringName) : Boolean = throw NotImplementedError("_propertyCanRevert is not implemented for Object")
+    open fun _propertyGetRevert(name: StringName): Any = throw NotImplementedError("_propertyGetRevert is not implemented for Object")
+    open fun _set(name: StringName, value: Any) : Unit = throw NotImplementedError("_set is not implemented for Object")
+    open fun _toString(): String = throw NotImplementedError("_toString is not implemented for Object")
```

Comment:
Shouldn't the default delegate to the regular `toString` method instead?

In the current state, doesn't that crash when one tries to print a jvm object from gdscript for example and does not override this method?
or does it also need to be registered for godot to be called in the first place like the rest?

Reply:
The way it works.
The native C++ implementation of `toString()` is the one that call `_toString()` IF it's implemented in the script, in the sense it is registered. It's the same for all case of `method()` and `_method()`.
Right now the JVM side just have a dummy implementation, so the class is not abstract and can be override but the script itself is not aware of it.
The user is then supposed to override it to make it useable.